Docs/KDESyntaxDefinition/dafny.xml

```diff
@@ -0,0 +1,151 @@
+  <highlighting>
+    <list name="keywords">
+      <item>abstract</item>
+      <item>allocated</item>
+      <item>as</item>
+      <item>assert</item>
+      <item>assume</item>
+      <item>break</item>
+      <item>by</item>
+      <item>calc</item>
+      <item>case</item>
+      <item>class</item>
+      <item>codatatype</item>
+      <item>colemma</item>
+      <item>const</item>
+      <item>constructor</item>
+      <item>copredicate</item>
+      <item>datatype</item>
+      <item>else</item>
+      <item>exists</item>
+      <item>expect</item>
+      <item>export</item>
+      <item>extends</item>
+      <item>false</item>
+      <item>forall</item>
+      <item>fresh</item>
+      <item>function</item>
+      <item>ghost</item>
+      <item>if</item>
+      <item>import</item>
+      <item>in</item>
+      <item>include</item>
+      <item>inductive</item>
+      <item>iterator</item>
+      <item>label</item>
+      <item>lemma</item>
+      <item>match</item>
+      <item>method</item>
+      <item>modify</item>
+      <item>module</item>
+      <item>new</item>
+      <item>newtype</item>
+      <item>null</item>
+      <item>old</item>
+      <item>opened</item>
+      <item>predicate</item>
+      <item>print</item>
+      <item>protected</item>
+      <item>refines</item>
+      <item>return</item>
+      <item>returns</item>
+      <item>reveal</item>
+      <item>static</item>
+      <item>then</item>
+      <item>this</item>
+      <item>trait</item>
+      <item>true</item>
+      <item>twostate</item>
+      <item>type</item>
+      <item>unchanged</item>
+      <item>var</item>
+      <item>where</item>
+      <item>while</item>
+      <item>yield</item>
+      <item>yields</item>
+    </list>
+    <list name="types">
+      <item>array</item>
+      <item>bool</item>
+      <item>char</item>
+      <item>imap</item>
+      <item>int</item>
+      <item>iset</item>
+      <item>map</item>
+      <item>multiset</item>
+      <item>nat</item>
+      <item>object</item>
+      <item>real</item>
+      <item>seq</item>
+      <item>set</item>
+      <item>string</item>
+    </list>
+
+    <list name="verification keywords">
+      <item>decreases</item>
+      <item>ensures</item>
+      <item>invariant</item>
+      <item>modifies</item>
+      <item>provides</item>
+      <item>reads</item>
+      <item>requires</item>
+      <item>reveals</item>
+    </list>
+
+    <contexts>
+      <context name="Normal" attribute="Normal Text" lineEndContext="#stay">
+        <keyword attribute="Keyword" String="keywords" context="#stay" />
+        <keyword attribute="Types" String="types" context="#stay" />
+        <keyword attribute="Assertions" String="verification keywords" context="#stay" />
+
+        <RegExpr attribute="Types" String="bv0|bv[1-9]\d*"/>
+        <RegExpr attribute="Types" String="array[2-9]\d*"/>
```

Comment:
This doesn't include types like `array10` and `array150`. Is the following legal KDE syntax syntax?

    "array[2-9]\d*|array1\d+"

Reply:
Ah good catch - none of those were in the Dafny test files I tried it on. Will fix.

Reply:
Fixed this in the merged version